picard/config.py

```diff
@@ -34,8 +34,19 @@ class ConfigSection(LockableObject):
 
     def __init__(self, config, name):
         super().__init__()
-        self.__config = config
+        self.__qt_config = config
+        self.__config = {}
         self.__name = name
+        self.load_keys()
+
+    def __qt_keys(self):
+        prefix = '%s/' % self.__name
```

Comment:
Instead of using % format, since key building here are very simple, using concatenation will waste less cpu cycles.
Since config options can be read in loops, optimization makes sense, and doesn't cost much in readibility.

```python
prefix = '%s/' % self.__name
key = "%s/%s" % (self.__name, name)
```

vs

```python
prefix = self.__name + '/'
key = self.__name + '/' + name
```

```python
from timeit import timeit

runs = 10000000


def print_results(time, start_string):
    print('%s\nTotal: %.4f\nAvg: %.4fns\n' % (start_string, time,
                                              (time/runs)*1000000000))

t1 = timeit('"%s/%s" % (a, b)',
            setup='a="xxx"; b="yyy"',
            number=runs)
t2 = timeit("a + '/' + b",
            setup='a="xxx"; b="yyy"',
            number=runs)
t3 = timeit('a + "/" + b',
            setup='a="xxx"; b="yyy"',
            number=runs)
t4 = timeit('"{}/{}".format(a, b)',
            setup='a="xxx"; b="yyy"',
            number=runs)

print_results(t1, '% replacement')
print_results(t2, 'concatenation single quotes')
print_results(t3, 'concatenation double quotes')
print_results(t4, '.format method')
```

```
% replacement
Total: 3.1121
Avg: 311.2064ns

concatenation single quotes
Total: 1.6318
Avg: 163.1769ns

concatenation double quotes
Total: 1.7592
Avg: 175.9204ns

.format method
Total: 4.3462
Avg: 434.6209ns
```